app/presenters/section_presenter.rb

```diff
@@ -29,8 +29,9 @@ def render_govspeak(body)
   end
 
   def render_footnotes_heading(body)
-    FootnotesSectionHeadingRenderer.create.call(
-      OpenStruct.new(body: body)
-    ).body
+    footnote_open_tag = '<div class="footnotes">'
+    heading_tag = '<h2 id="footnotes">Footnotes</h2>'
+
+    body.gsub(footnote_open_tag, "#{heading_tag}\\&")
   end
 end
```

Comment:
Minor: I'd be tempted to squash this commit with the subsequent one that removes the `FootnotesSectionHeadingRenderer` and its spec.